### PR TITLE
chore(CI): add OBS workflows

### PR DESCRIPTION
**Pull request by tsic404**
add OBS workflows and modify PKGBUILD support OBS build

log:

### CODE REVIEW

**Comment by tsic404** [comment]
开启Debian Sid [amd64] deepin_develop(23) [ amd64, arm64], archlinux [ amd64 ] 构建

**Comment by tsic404** [comment]
https://build.deepin.com/package/show/deepin:Develop:dde/qt5integration
qt5integration 是 native package, gbp 默认压缩为 .tar.xz, 解压后直接为路径 `pacakge`

**Review by kegechen — APPROVED** [review]

File: .obs/workflows.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        source_project: deepin:Develop:dde
+        source_package: %{SCM_REPOSITORY_NAME}
+        target_project: deepin:Unstable:dde
+  filters:
+    event: tag_push
+
+commit_build:
+  steps:
+    - trigger_services:
+        project: ddeepin:Develop:dde
```
Comment (kegechen) [inline .obs/workflows.yml:48]: 果然这里也有写错的。幸好看了一眼 https://github.com/linuxdeepin/dde-app-services/pull/42

**Review by kegechen — CHANGES_REQUESTED** [review]

File: .obs/workflows.yml (hunk 1 above)
Comment (tsic404) [reply .obs/workflows.yml]: 已更正

**Review by kegechen — APPROVED** [review]

**Pull request merged**